docs/Contributors-Guide/Bug-Triage.md

```diff
@@ -272,31 +272,29 @@ During triaging you might come across a particular bug which is present
 across multiple version of GlusterFS. Here are the course of actions:
 
 -   We should have separate bugs for each release (We should
-    clone bugs if required)
--   Bugs in released versions should be depended on bug for mainline
-    (master branch) if the bug is applicable for mainline.
-    -   This will make sure that the fix would get merged in master
-        branch first then the fix can get ported to other stable
-        releases.
+    clone bugs if required), the bug needs to be fixed.
+
+-   Bug reported for any specific version neednot be cloned for
+    'mainline', as by default, as a policy, we need to submit the fix
+    for 'master' branch before backporting fix to specific branch.
+    Only when a bug needs to be fixed in multiple 'released' versions,
+    then there is a clone requried.
 
 *Note: When a bug depends on other bugs, that means the bug cannot be
 fixed unless other bugs are fixed (depends on), or this bug stops other
 bugs being fixed (blocks)*
 
 Here are some examples:
 
--   A bug is raised for GlusterFS 3.5 and the same issue is present in
-    mainline (master branch) and GlusterFS 3.6
-    -   Clone the original bug for mainline.
-    -   Clone another for 3.6.
-    -   And have the GlusterFS 3.6 bug and GlusterFS 3.5 bug 'depend on'
-        the 'mainline' bug
+-   A bug is raised for GlusterFS 3.12 and the same issue is present in
+    mainline (master branch) and GlusterFS 4.1.
+    -   See if the bug is of severity 'Urgent'/'High', and only if yes,
+        clone it for 4.1
```

Comment:
These examples make a little more clear what was described above. Please add an explanation which bug should be in the 'blocks' and which bug should be in the 'depends on' fields.

Reply:
Done